resources/eds_templates/diff_report/templates/macros.html

```diff
@@ -0,0 +1,57 @@
+{# Default way of showing errors #}
```

Comment:
we need to investigate how can we provide mactos libraries to the eds4jinja reports. So we no longer carry this macros file around. New jira ticket I guess